crates/util/python-utils/src/schnorr.rs

```diff
@@ -0,0 +1,62 @@
+use std::str::FromStr;
+
+use pyo3::{pyfunction, PyResult};
+use secp256k1::{schnorr::Signature, Keypair, Message, SecretKey, SECP256K1};
+use strata_crypto::{
+    sign_schnorr_sig as schnorr_sig_sign, verify_schnorr_sig as schnorr_sig_verify,
+};
+use strata_primitives::buf::{Buf32, Buf64};
+
+/// Signs a message using the Schnorr signature scheme.
+///
+/// This function generates a Schnorr signature for the given message using the provided secret key.
```

Comment:
```suggestion
/// Generates a Schnorr signature for the given message using the provided secret key.
```

"Function" is not needed. This is a `fn` anyways.